src/index.js

```diff
@@ -1,27 +1,74 @@
 import './style.css';
 
 const todoList = document.querySelector('.todo-list');
-const todos = [
-  {
-    index: 1,
-    completed: true,
-    task: 'Wash the dish',
-  },
-  {
-    index: 2,
-    completed: true,
-    task: 'Go to the gym',
-  },
-];
+const clearBtn = document.querySelector('.clear-btn');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+function saveTasks() {
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+function removeTask(id) {
+  todos = todos.filter((task) => task.id !== id);
+  saveTasks();
+}
 
 function displayTasks() {
-  todos.forEach((todos) => {
+  todoList.innerHTML = '';
+
+  todos.forEach((task) => {
     const li = document.createElement('li');
     li.className = 'list-item';
+    li.innerHTML = `
+      <input type="checkbox" class="check" />
+      ${task.task}
+      <span class="material-icons close" data-id="${task.id}">close</span>
+    `;
+    todoList.appendChild(li);
+  });
 
-    li.innerHTML = `<input type="checkbox" class="check" /> ${todos.task} <span class="material-icons" id="close">close</span>`;
+  const closeButtons = document.querySelectorAll('.close');
+  closeButtons.forEach((button) => {
+    button.addEventListener('click', (e) => {
+      const taskId = parseInt(e.target.dataset.id, 10);
+      removeTask(taskId);
+      displayTasks();
```

Comment:
Good job creating the functionality to add and remove, however according to the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Create%20a%20new%20JavaScript%20file%20for%20the%20new%20functionality.), this functions should be added to a new file, consider doing the refactor in order to meet the milestone requirements.